classes/class-wc-connect-settings-pages.php

```diff
@@ -80,10 +81,13 @@ public function output_shipping_settings_screen() {
 				<?php
 			}
 
-			$extra_args        = array();
+			$extra_args = array(
+				'live_rates_types' => $this->service_schemas_store->get_all_shipping_method_ids(),
+			);
+
 			$carriers_response = $this->api_client->get_carrier_accounts();
 			if ( ! is_wp_error( $carriers_response ) && $carriers_response ) {
-				$extra_args['carriers'] = $carriers_response->carriers;
```

Comment:
Changed argument `carriers` to `carrier_accounts`, to generate less possible confusion when the live rates' "Carriers" are also displayed